mail-client/balsa/balsa-2.6.4.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome2-utils meson xdg
+
+DESCRIPTION="Email client for GNOME"
+HOMEPAGE="http://pawsa.fedorapeople.org/balsa/"
+SRC_URI="http://pawsa.fedorapeople.org/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~x86"
+
+IUSE="gnome +gnome-keyring kerberos ldap rubrica sqlite systray webkit xface"
+
+# TODO: internal spell checking via enchant-2 instead of gtkspell/gspell?
+DEPEND="
+	>=dev-libs/glib-2.48.0:2
+	>=x11-libs/gtk+-3.24.0:3
+	>=dev-libs/gmime-3.2.6:3.0
+	>=net-libs/gnutls-3.0:=
+	dev-libs/fribidi
+	>=dev-libs/libical-2.0.0:=
+	webkit? (
+		>=net-libs/webkit-gtk-2.28.0:4
+		>=dev-db/sqlite-3.24:=
+		app-text/html2text
+	)
+	>=app-crypt/gpgme-1.13.0:=
+	sqlite? ( >=dev-db/sqlite-3.24:= )
+	ldap? ( net-nds/openldap:= )
+	rubrica? ( dev-libs/libxml2:2 )
+	kerberos? ( app-crypt/mit-krb5 )
+	xface? ( >=media-libs/compface-1.5.1:= )
+	gnome? ( x11-libs/gtksourceview:4 )
+	media-libs/libcanberra:=[gtk3]
+	gnome-keyring? ( app-crypt/libsecret )
+	>=app-text/gspell-1.6:0=
+
+	net-mail/mailbase
+	x11-themes/hicolor-icon-theme
+	x11-themes/adwaita-icon-theme
+	dev-libs/openssl:0=
+	systray? ( x11-libs/xapp )
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	dev-util/gtk-update-icon-cache
+	dev-util/intltool
+	dev-util/itstool
+	virtual/pkgconfig
+	sys-devel/gettext
+	dev-libs/libxml2:2
+"
+
+# DOCS="AUTHORS ChangeLog HACKING NEWS README TODO docs/*"
+
+src_prepare() {
+	default
+	sed -i 's/if gtk_update_icon_cache_program.found()/if false/' images/meson.build || die 
+}
+
+src_configure() {
+	local myconf=(
```

Comment:
`meson_src_configure` looks for `emesonargs`, so none of this is working.

Reply:
Oh that is the reason why the build worked in meson: I never passed the options to it. WIthout that it uses -Dhtml-widged=no as the default, which causes the build to fail. 
I corrected the emesonargs and pushed a version that sets -Dhtml-widget=webkit2 unconditionally.